firefox-ios/Client/Frontend/Library/Bookmarks/BookmarksViewController.swift

```diff
@@ -81,6 +81,14 @@ class BookmarksViewController: SiteTableViewController,
         return button
     }()
 
+    private lazy var emptyStateView: BookmarksFolderEmptyStateView = {
```

Comment:
we should prefer the `.build` method all over for consistency, especially for newly added code.